### PR TITLE
Update CSS at-rule hints

### PR DESCRIPTION
**Pull request by valtlai**
Add `@counter-style` and `@font-feature-values` at-rules.
Reorder the rules alphabetically.

### CODE REVIEW

File: src/extensions/default/CSSAtRuleCodeHints/AtRulesDef.json [hunk 1]
```diff
@@ -1,10 +1,12 @@
...
-    "@namespace": "Tells the CSS engine that all its content must be considered prefixed with an XML namespace.",
-    "@media":     "A conditional group rule which will apply its content if the device meets the criteria of the condition defined using a media query.",
-    "@supports":  "A conditional group rule which will apply its content if the browser meets the criteria of the given condition.",
-    "@page":      "Describes the aspect of layout changes which will be applied when printing the document.",
-    "@font-face": "Describes the aspect of an external font to be downloaded.",
-    "@keyframes": "Describes the aspect of intermediate steps in a CSS animation sequence."
+    "@charset":             "Defines the character set used by the style sheet.",
+    "@counter-style":       "Defines specific counter styles that are not part of the predefined set of styles.",
+    "@font-face":           "Describes the aspect of an external font to be downloaded.",
+    "@font-feature-values": "Define common names in font-variant-alternates for feature activated differently in OpenType.",
```
Comment (petetnt) [inline src/extensions/default/CSSAtRuleCodeHints/AtRulesDef.json]: [MDN}(https://developer.mozilla.org/en-US/docs/Web/CSS/%40font-feature-values) phrases this as `Defines a common name in font-variant-alternates for features activated differently in OpenType.`, not quite sure which one is more correct though 
¯\\\_(ツ)_/¯

Comment (valtlai) [reply src/extensions/default/CSSAtRuleCodeHints/AtRulesDef.json]: Hmm, these descriptions are from [another MDN page](https://developer.mozilla.org/en-US/docs/Web/CSS/At-rule).

Comment (petetnt) [reply src/extensions/default/CSSAtRuleCodeHints/AtRulesDef.json]: Then it's good enough for me (maybe fix them in MDN too? 😂 )

**Review by petetnt — APPROVED** [review]

File: src/extensions/default/CSSAtRuleCodeHints/AtRulesDef.json (hunk 1 above)
Comment (valtlai) [reply src/extensions/default/CSSAtRuleCodeHints/AtRulesDef.json]: Made it a bit more consistent by adding *-s*. 😇

**Review by ficristo — CHANGES_REQUESTED** [review]
Fix the test.
Otherwise LGTM.

**Comment by valtlai** [comment]
Fixed the test.

**Review by ficristo — APPROVED** [review]

**Comment by ficristo** [comment]
Thank you!

**Pull request merged**